spec/parser_spec.rb

```diff
@@ -1,16 +1,23 @@
 require_relative '../lib/parser'
 
-describe '#get_first_line' do
-  context 'if file exists' do
-    it 'and matches error' do 
-      expect { Parser.new('any.log') }.to raise_error("Arquivo não encontrado")
+describe Parser do
+  describe 'file does not exist' do
+    it 'and matches error' do
+      expect { Parser.new('any.log') }.to raise_error('Arquivo não encontrado')
     end
   end
 
-  context 'first line' do
-    it 'matches string in file' do 
-        data = Parser.new('games.log')
-        expect(data.get_first_line).to match("  0:00 ------------------------------------------------------------\n")
+  describe 'first line' do
```

Comment:
```suggestion
  describe '#first_line' do
```

https://www.betterspecs.org/#describe